core/server/master/src/main/java/alluxio/web/WebInterfaceConfigurationServlet.java

```diff
@@ -16,11 +16,8 @@
 import alluxio.master.file.FileSystemMaster;
 
 import com.google.common.collect.Sets;
-
-import org.apache.commons.lang3.tuple.ImmutablePair;
-import org.apache.commons.lang3.tuple.Pair;
-
-import com.google.common.collect.Sets;
+import org.apache.commons.lang3.tuple.ImmutableTriple;
+import org.apache.commons.lang3.tuple.Triple;
```

Comment:
Related to this PR: we should probably come up with something like a delimiter and just pass a String